digits/utils/filesystem.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2014-2015, NVIDIA CORPORATION.  All rights reserved.
+
+import os.path
+
+def sizeof_fmt(num, suffix='B'):
+    """
+    return human-readable size
+    """
+    for unit in ['','K','M','G','T','P','E','Z']:
+        if abs(num) < 1024.0:
+            if unit != '':
+                return "%3.1f%s%s" % (num, unit, suffix)
+            else:
+                return "%d%s" % (num, suffix)
+        num /= 1024.0
+    return "%.1f%s%s" % (num, 'Y', suffix)
```

Comment:
Is this just a refactoring of `digits.utils.sizeof_fmt` ([here](https://github.com/NVIDIA/DIGITS/blob/v2.1.0/digits/utils/__init__.py#L100-L106))?